pkg/watch/mux.go

```diff
@@ -149,6 +151,7 @@ func (m *Broadcaster) Action(action EventType, obj runtime.Object) {
 // You must not call Action after calling Shutdown.
```

Comment:
Clarification: distributed to, but not necessarily handled by, all extant watchers; stuff could remain sitting in the outbound channels.